brainfuck/brainfuck_test.go

```diff
@@ -0,0 +1,152 @@
+package brainfuck
```

Comment:
here you should test your public API + some stuff only for `brainfuck.go` file. Everything else should be in the test files for appropriate functionality.